main.py

```diff
@@ -25,9 +24,13 @@ def if_statements(directory_path):
 
 
 @app.command()
-def looping_constructs():
+def looping_constructs(directory_path):
     """Determine number of looping constructs in a Python directory."""
-    print("In progress")
+    cast_dict = generate_trees(directory_path)
+    while_loops_dict = counter.count_whileloops(cast_dict)
+    for_loops_dict = counter.count_forloops(cast_dict)
```

Comment:
This command should be returning the number of for loops, number of while loops, and those numbers combined. I believe it should look like this:

```
def looping_constructs(directory_path):
    """Determine number of looping constructs in a Python directory."""
    cast_dict = generate_trees(directory_path)
    while_loops_dict = counter.count_whileloops(cast_dict)
    for_loops_dict = counter.count_forloops(cast_dict)
    number_for_loops = counter.amount_loops(for_loops_dict)
    number_while_loops = counter.amount_loops(while_loops_dict)
    total_loops = number_for_loops + number_while_loops
    print("Number for loops: " + number_for_loops)
    print("Number while loops: " + number_while_loops)
    print("Number total looping constructs: " + total_loops)
```